torchmd/utils.py

```diff
@@ -72,3 +73,40 @@ def save_argparse(args, filename, exclude=None):
                 if k is exclude:
                     continue
                 f.write(f"{k}={v}\n")
+
+
+#%% Converter
+# fmt: off
+
+elements=["A", "H", "He", "Li", "Be", "B", "C", "N", "O", "F", "Ne", "Na",
```

Comment:
is this the periodic table? you can get it from
```py
from moleculekit.periodicttable import periodictable_by_number
print(periodictable_by_number[6].symbol) # C
```
instead of redifining it here

Reply:
Actually don't worry, I'll make the change

Reply:
Perfect